docs/assets/simple_execution_modes.drawio.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Random thought: Maybe split the yellow  OoO plugins box into two boxes to avoid that people think it is a single process for all OoO plugins? I know it shows 2 WPEProcesses, but it might make it more clear.